buildout.cfg

```diff
@@ -62,14 +62,6 @@ eggs = i18ndude
 # Don't use a released version of ploneorg.addonlisting
 ploneorg.addonlisting =
 
-setuptools = 24.3.0
```

Comment:
Those Pins are not necessary or did even harm the tests.

As this is not the production buildout, that does not matter much.